mappings/net/minecraft/network/packet/s2c/play/ProfilelessChatMessageS2CPacket.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_7827 net/minecraft/network/packet/s2c/play/ProfilelessChatMessageS2CPacket
```

Comment:
The term 'profileless' is a bit odd but I'm not familiar enough with the code to suggest an improvement.

Reply:
This is used for chat messages sent through command in a way that makes it unverifiable - such as `/execute as entity` or command block. They are not associated with a sender profile, thus its name. (Is in use since 1.19, so not a new name)